src/components/Modal.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div id="modal">
+    <div id="header">
+      <div id="title">
+        <p class="title">Detalhes do usuário</p>
+      </div>
+      <div id="headerBtn">
+        <button class="btn click" @click="closeModal">
+          <i class="btn-ic-close fa-regular fa-x"></i>
+        </button>
+      </div>
+    </div>
+    <div id="contents">
+      <div id="rightColumn">
+        <div id="cel">
+          <label>Celular</label>
+          <p class="dados">{{ dados.phone | VMask("(##) # ####-####") }}</p>
+        </div>
+        <div id="ctt">
+          <label>Contato</label>
+          <div id="contact">
+            <div id="cttSms" v-if="contact">
+              <i class="fa-solid fa-message"></i>
+              <p class="dados">Sms e email</p>
+            </div>
+            <div id="cttWhatsApp" v-else>
+              <i class="fa-brands fa-whatsapp"></i>
+              <p class="dados">WhatsApp</p>
+            </div>
+          </div>
+        </div>
+        <div id="email">
+          <label>Email</label>
+          <p class="dados">{{ dados.email }}</p>
+        </div>
+      </div>
+      <div id="leftColumn">
+        <div id="cpf">
+          <label>CPF</label>
+          <p class="dados">{{ dados.cpf | VMask("###.###.###-##") }}</p>
+        </div>
+        <div id="name">
+          <label>Name</label>
+          <p class="dados">{{ dados.fullname }}</p>
+        </div>
+        <div id="date">
+          <label>Data</label>
+          <p class="dados">{{ dados.birthDate | VMask("####/##/##") }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Modal",
+  props: {
+    dados: {
+      type: Object,
+    },
+  },
+  computed: {
+    contact() {
+      if (this.dados.contacts) {
+        return true;
+      } else {
+        return false;
+      }
```

Comment:
Conseguimos fazer assim?
```suggestion
      return Boolean(this.dados.contacts);
```